pull_chart_tarball.go

```diff
@@ -50,7 +50,7 @@ func (c *Client) doFile(ctx context.Context, req *http.Request) (string, error)
 			// Github pages 404 produces full HTML page which
 			// obscures the logs.
 			if resp.StatusCode == http.StatusNotFound {
-				return microerror.Maskf(executionFailedError, fmt.Sprintf("got StatusCode %d for url %#q", resp.StatusCode, req.URL.String()))
+				return backoff.Permanent(microerror.Maskf(executionFailedError, fmt.Sprintf("got StatusCode %d for url %#q", resp.StatusCode, req.URL.String())))
```

Comment:
This makes total sense and I think a 404 is more likely. As it can happen if you choose the wrong version. 

For the host we always use GitHub at the moment.